src/main/java/alexiil/mc/lib/net/impl/CoreMinecraftNetUtil.java

```diff
@@ -125,46 +121,39 @@ public static ActiveMinecraftConnection getConnection(PlayerEntity player) {
     }
 
     public static void load() {
-        if (PacketContext.class.isAssignableFrom(ServerPlayNetworkHandler.class)) {
-            // Fabric API: Networking V0
-            ServerSidePacketRegistry.INSTANCE.register(ActiveMinecraftConnection.PACKET_ID, (ctx, buffer) -> {
-                onServerReceivePacket((ServerPlayNetworkHandler) ctx, NetByteBuf.asNetByteBuf(buffer));
-            });
-        } else {
-            // Fabric API: Networking V1
-            // Object and then cast to avoid the verifier classloading
-            Object handler = new ServerPlayNetworking.PlayChannelHandler() {
-                @Override
-                public void receive(
-                    MinecraftServer server, ServerPlayerEntity player, ServerPlayNetworkHandler handler,
-                    PacketByteBuf buf, PacketSender responseSender
-                ) {
-                    ActiveServerConnection connection = getServerConnection(handler);
-                    if (connection == null) {
-                        return;
-                    }
-
-                    NetByteBuf b = NetByteBuf.asNetByteBuf(buf.copy());
-                    server.execute(() -> {
-                        try {
-                            connection.onReceiveRawData(b);
-                            b.release();
-                        } catch (InvalidInputDataException e) {
-                            e.printStackTrace();
-                            handler.disconnect(
-                                new LiteralText("LibNetworkStack: read error (see server logs for more details)\n" + e)
-                            );
-                        }
-                    });
+        // Fabric API: Networking V1
+        // Object and then cast to avoid the verifier classloading
```

Comment:
Just FYI this was to work around networking v1 potentially not being present.

Reply:
Yeah, I figured that the deprecation of v0 would mean v1 would always be present.